### ISSUE TITLE
[bug] eReader address does not accept multiple comma-separated emails

### ISSUE THREAD

**Issue by Jerdle** [crocodilestick/Calibre-Web-Automated#154]
**Describe the bug**
Title. I can't add multiple email addresses to the 'Send to eReader Email Address' field for a user.

**To Reproduce**
1. Edit a user
2. Add multiple comma-separated emails to the field such as 'email@email.com,gmail@gmail.com'
3. Save
4. Seems the last email specified is what's accepted

**Expected behavior**
Need to add multiple emails

**Screenshots**
![Screenshot 2024-12-22 at 9 39 05 PM](https://github.com/user-attachments/assets/bedcc040-1f6b-4e72-be19-4eab17294e7f)
After save
![Screenshot 2024-12-22 at 9 39 27 PM](https://github.com/user-attachments/assets/b23f29f5-71a1-4ea1-8d83-5e9898c97fb7)

**Configuration(please complete the following information):**
Synology NAS in docker
![Screenshot 2024-12-22 at 9 43 19 PM](https://github.com/user-attachments/assets/40772b84-b66b-4f5e-92e1-10f12371f7a9)

**Comment by nolij420** [comment]
FYI: It looks like Calibre-Web project had this same issue
https://github.com/janeczku/calibre-web/issues/3130

**Comment by crocodilestick** [comment]
Yeah it's been fixed, is available in the dev build and will be in the next release

-------- Original Message --------

**Comment by crocodilestick** [comment]
Fixed as of V3.0.1
